doc/v2/howto/rnn/recurrent_group_en.md

```diff
@@ -1,3 +1,96 @@
 # Recurrent Group Tutorial
 
-TBD
+## Overview
+
+Sequential data is a major input type dealing with natural language processing tasks.
+
+A sentence is a sequence of words. Many sentences form a paragraph further. Therefore, a paragraph can be viewed as a nested sequence with two level, where each element of the sequence is another sequence.
+
+The double-level sequence is a very flexible data organization method supported by PaddlePaddle, which helps us to better describe more complex language data such as discribing paragraphs and several rounds of dialogues. Based on double-level sequence input, we can design and build a flexible, hierarchical RNN model that encodes input data from the word and sentence level, and can also introduce more complex memory mechanisms to better complete some complex language understanding tasks.
```

Comment:
I think this paragraph could be shorter and bring the readers' focus to two-level sequences while keeping them informed that we do have the solution to arbitrary level data.

> PaddlePaddle and PaddlePaddle v2 support two-level recursive sequential data. For the support of arbitrary levels, please refer to PaddlePaddle Fluid.

Reply:
OK, imply PaddlePaddle Fluid for two-level sequence